client/src/EMS/BloodPressureField.js

```diff
@@ -6,6 +6,7 @@ import FormInput from '../Components/FormInput';
 import { useForm } from '../Components/Form';
 
 import './BloodPressureField.scss';
```

Comment:
Can you delete this and the scss file?  The new component handles all of the styling, I believe.